llmware/parsers.py

```diff
@@ -3066,6 +3066,151 @@ def basic_library_duplicate_check(self, fn):
 
         return in_library
 
+    def parse_delimiter_config(self,fp, fn, cols=None, mapping_dict=None, delimiter=","):
+
+        """ Designed for intake of a 'pseudo-db csv table' and will add rows to library with mapped keys.
```

Comment:
Please change the docstring to the standard we use, see e.g. https://github.com/llmware-ai/llmware/blob/bfec79b2ee570f006e0a4dceb524c4d9bfe67f95/llmware/embeddings.py#L98-L114

Reply:
Hello MacOS, you to be blunt ;) this is the original Code from llmware, all I did was delete one line and add abouzt 5 lines :P all the rest is the pure original :D

Reply:
I'm sorry about being blunt, but this here is a code review and hence meant to be short and to the point. My intention is not to be rude. ;)

``` this is the original Code from llmware, all I did was delete one line and add abouzt 5 lines :P all the rest is the pure original :D```
Please see my comments in the main conversations pane about that.